Classes/LinphoneUI/UIVideoButton.m

```diff
@@ -95,6 +95,10 @@ - (bool)onUpdate {
 	bool camera_enabled = false;
 	LinphoneCore *lc = [LinphoneManager getLc];
 	LinphoneCall *currentCall = linphone_core_get_current_call(lc);
+    if(linphone_call_get_state(currentCall) == LinphoneCallOutgoingRinging){
+        camera_enabled = TRUE;
```

Comment:
Stop scaring the deaf community that their video isn't on while the call is ringing.